en/config/actuators.md

```diff
@@ -0,0 +1,116 @@
+# Actuator Configuration and Testing
+
+After the airframe is setup you should configure the geometry, assign actuators to outputs, and test the actuator response.
```

Comment:
Just FMI What do you mean by "is setup". I think you mean after you have wired up your airframe and selected the airframe type? I.e. I'm trying to confirm the order in which you can do this w.r.t. other config sections and I think it can be done after firmware and airframe selection

Reply:
> I think you mean after you have wired up your airframe and selected the airframe type?

Correct

> Further, is all of this a "have to". Specifically are their airframes where you will arrive here, having selected your airframe, and all you need to do is test that the motors spin in the right way?

Yes for fully preconfigured off-the shelf vehicles that is correct. For generic airframes the outputs are most likely not assigned. In that case the UI tells you that you still need to assign them:
![image](https://user-images.githubusercontent.com/281593/146323145-359e1f28-2681-49b8-842b-6287779a38f8.png)